### PR TITLE
Adding apacheds/1.5.7.wso2v3 and  nimbuds/2.26.1.wso2v3

### PR DESCRIPTION
**Pull request by hpmtissera**

### CODE REVIEW

File: nimbus/2.26.1.wso2v3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    <name>WSO2 Carbon Orbit - Nimbus JWT Library</name>
+    <version>2.26.1.wso2v3</version>
+    <description>
+        This bundle will export packages from nimbus-jose-jwt-2.26.1.jar
+    </description>
+    <url>http://wso2.org</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.nimbusds</groupId>
```
Comment (arunasujith) [inline nimbus/2.26.1.wso2v3/pom.xml]: Maven project groupId should start with org.wso2.*

File: apacheds/1.5.7.wso2v3/pom.xml [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.directory.wso2</groupId>
```
Comment (arunasujith) [inline apacheds/1.5.7.wso2v3/pom.xml]: Maven project groupId should start with org.wso2.*

File: apacheds/1.5.7.wso2v3/pom.xml [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+    <artifactId>apacheds</artifactId>
+    <packaging>bundle</packaging>
+    <name>ApacheDirectory</name>
+    <version>1.5.7.wso2v3</version>
+    <description>A custom wso2 products or solution which wraps Apache DS</description>
+    <url>http://www.wso2.com</url>
+
+    <dependencies>
```
Comment (arunasujith) [inline apacheds/1.5.7.wso2v3/pom.xml]: Make all dependencies optional

File: apacheds/1.5.7.wso2v3/pom.xml [hunk 4]
```diff
@@ -0,0 +1,519 @@
+<!--
+ ~ Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (sameera-jayasoma) [inline apacheds/1.5.7.wso2v3/pom.xml]: We don't put the range here. Just put 2015

File: apacheds/1.5.7.wso2v3/pom.xml [hunk 5]
```diff
@@ -0,0 +1,519 @@
...
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.apache.directory</groupId>
```
Comment (sameera-jayasoma) [inline apacheds/1.5.7.wso2v3/pom.xml]: groupID is wrong. use the new groupID naming policy

File: apacheds/1.5.7.wso2v3/pom.xml [hunk 6]
```diff
@@ -0,0 +1,519 @@
...
+                        <Import-Package>
+                            antlr.*,
+			                org.slf4j.*;version="${slf4j.osgi.version}",
+                            org.apache.commons.lang.*;version="${commons.lang.osgi.version}",
+                            org.apache.commons.io.*;version="${commons.io.osgi.version}",
+                            org.apache.commons.collections.*;version="${commons.collections.osgi.version}",
+                            *;resolution:=optional
+                        </Import-Package>
+                        <Embed-Dependency>
+                            bcprov-jdk14|mina-core|
```
Comment (sameera-jayasoma) [inline apacheds/1.5.7.wso2v3/pom.xml]: Why are we embedding so many dependencies here? This is a bad smell

File: nimbus/2.26.1.wso2v3/pom.xml [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.nimbusds</groupId>
```
Comment (sameera-jayasoma) [inline nimbus/2.26.1.wso2v3/pom.xml]: GroupID is wrong

File: apacheds/1.5.7.wso2v3/pom.xml [hunk 8]
```diff
@@ -0,0 +1,520 @@
...
+  ~  KIND, either express or implied.  See the License for the
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.apache.directory</groupId>
```
Comment (kasunbg) [inline apacheds/1.5.7.wso2v3/pom.xml]: The groupId should be "org.wso2.orbit.org.apache.directory"

File: nimbus/2.26.1.wso2v3/pom.xml [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.nimbusds</groupId>
```
Comment (kasunbg) [inline nimbus/2.26.1.wso2v3/pom.xml]: wrong groupId. it should be org.wso2.orbit.com.nimbusds

**Pull request merged**